src/main/java/com/iridium/iridiumskyblock/configs/Schematics.java

```diff
@@ -62,10 +62,12 @@
     public static class SchematicWorld {
         public XBiome biome;
         public String schematicID;
+        public double yHeightIsland;
 
-        public SchematicWorld(XBiome biome, String schematicID) {
+        public SchematicWorld(XBiome biome, String schematicID, double yheightisland) {
```

Comment:
Two issues:
1. Please user lowerCamelCase for variable names
2. I think the name `islandHeight` would be a better name. Height already implies that the y axis is meant.